src/Microsoft.ApplicationInsights/Managed/Shared/Extensibility/Implementation/Tracing/DiagnosticsTelemetryModule.cs

```diff
@@ -43,6 +42,16 @@ public DiagnosticsTelemetryModule()
         }
 
         /// <summary>
+        /// Gets or sets the delay between heartbeat emmissions in milliseconds.
```

Comment:
spelling: emissions
Suggest simplifying to "[..] delay between heartbeats in milliseconds."

Reply:
done